src/Microsoft.Framework.PropertyActivator.Internal/PropertyActivator.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using System.Reflection;
+
+namespace Microsoft.Framework.Internal
+{
+    public class PropertyActivator<TContext>
```

Comment:
This should have a test, if you don't want to do it now, log an issue (but better yet just do it)